README.md

```diff
@@ -1 +1,75 @@
-# hello-microverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Hello Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Hello Microverse ](#-hello-microverse-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [📝 License ](#-license-)
```

Comment:
Well done on using the README template provided by Microverse 💯 but several sections seem to have been removed from the `Table of Contents` 📄 these sections were required, only the `FAQ` is optional and can be removed. Make sure to add the other sections to make the README complete just like the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) 🤓

Reply:
Thank you.